src/main/scala/mesosphere/marathon/api/v2/GroupsResource.scala

```diff
@@ -61,13 +61,13 @@ class GroupsResource @Inject() (
         }
       }
       id match {
-        case ListApps(gid)              => groupResponse(gid.toRootPath, _.transitiveApps.map(V2AppDefinition(_)))
-        case ListRootApps()             => groupResponse(PathId.empty, _.transitiveApps.map(V2AppDefinition(_)))
+        case ListApps(gid)              => groupResponse(gid.toRootPath, _.transitiveApps)
+        case ListRootApps()             => groupResponse(PathId.empty, _.transitiveApps)
         case ListVersionsRE(gid)        => ok(jsonString(result(groupManager.versions(gid.toRootPath))))
         case ListRootVersionRE()        => ok(jsonString(result(groupManager.versions(PathId.empty))))
-        case GetVersionRE(gid, version) => groupResponse(gid.toRootPath, V2Group(_), version = Some(Timestamp(version)))
-        case GetRootVersionRE(version)  => groupResponse(PathId.empty, V2Group(_), version = Some(Timestamp(version)))
-        case _                          => groupResponse(id.toRootPath, V2Group(_))
+        case GetVersionRE(gid, version) => groupResponse(gid.toRootPath, { g => g }, version = Some(Timestamp(version)))
```

Comment:
@aquamatthias Do you think of a better way of defining a function which only forwards the group object?